toc/working-groups/app-runtime-interfaces.md

```diff
@@ -374,6 +374,7 @@ areas:
   - cloudfoundry/steno
   - cloudfoundry/runtimeschema
   - cloudfoundry/app-runtime-interfaces-infrastructure
+  - cloudfoundry/homebrew-tap
```

Comment:
Shouldn't this be in the `CLI` area, instead of `CAPI`?

Reply:
Sorry, my bad. fixed it.